config/game_options.txt

```diff
@@ -77,8 +77,8 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 ## Set to 0 to disable that mode.
 
 PROBABILITY TRAITOR 5
-PROBABILITY TRAITORBRO 2
-PROBABILITY TRAITORCHAN 4
+PROBABILITY TRAITORBRO 4
+PROBABILITY TRAITORCHAN 2
 PROBABILITY INTERNAL_AFFAIRS 3
```

Comment:
Consider turning IA down to 1. It is not a fun gamemode and honestly it takes away from the paranoia.

Reply:
What the fuck is internal affairs?

Reply:
Double Agents (Traitor Deathmatch) but with objectives not to break things or cause collateral damage on top of having implanted target trackers. Aka it's a game where the antags kill each other and avoid involving the crew in any way shape or form. Aka they don't even do their jobs as antags to make the round interesting.

It's a shitty gamemode. I want to disable it entirely honestly.

Reply:
redundant